src/stacktrace_windows.cpp

```diff
@@ -23,6 +23,8 @@
 #include <mutex>
 #include <g3log/g3log.hpp>
 
+#pragma comment(lib, "dbghelp.lib")
```

Comment:
Yes, but this is more or less the windows idiomatic way. Boost relies on the same mechanism. These standard windows libraries are sometimes hard to chase down, which one to link. But, of course, if the documentation and the build config say what to link, it's not strictly necessary.